src/core/util/props.js

```diff
@@ -127,11 +127,48 @@ function assertProp (
       valid = assertedType.valid
     }
   }
+
+  function invalidTypeMessage () {
```

Comment:
`invalidTypeMessage` sounds to me more like a property, not a function. How about `getInvalidTypeMessage()`?

Reply:
@phanan, I agree, with a verb in front it sounds more correct, fixed it.